packages/nivo-line/src/LineSlicesItem.js

```diff
@@ -54,34 +54,44 @@ LineSlicesItem.propTypes = {
     hideTooltip: PropTypes.func.isRequired,
     isHover: PropTypes.bool.isRequired,
     theme: PropTypes.object.isRequired,
+    tooltip: PropTypes.element,
```

Comment:
Same here @jbenua, tooltip should also allow `PropTypes.func`, it even checks for it in line 84:
```
renderContent={
  typeof tooltip === 'function' ? tooltip.bind(null, { ...slice }) : null
}
```

Reply:
I've fixed this in #275.

Reply:
Thanks Raphaël, I will watch master/npm when its merged.